utils/single_test.py

```diff
@@ -3,18 +3,18 @@
 Convert a single line from test output to run single test command.
 
 E.g:
-    "2: test_download_package_via_metalink (tests.test_yum_package_downloading.TestCaseYumPackageDownloading) ... ok"
+    "test_compress_file (tests.test_misc.TestCaseMisc) ... ok"
 
 To:
-    tests/python/tests/test_yum_repo_downloading.py:TestCaseYumRepoDownloading.test_download_and_update_repo_01
+    "python3 -m unittest discover -s tests/python/ -p tests/test_misc.py -k test_compress_file"
```

Comment:
I'm not sure I understand what this script is for, but I question its usefulness, it probably took more time to port it than it actually saved. Is anyone using it? (I'd suggest asking first whether we can delete these things before exerting the effort? :slightly_smiling_face: )

Reply:
I personally have never used it and its not called from anywhere else in the repo.

Reply:
Ok, I'll just remove it.